docs/proposals/metrics-api.md

```diff
@@ -0,0 +1,134 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
```

Comment:
This approach will force a thorough API review of each and every field again, which has already been done on the previous proposal :(